harmony/README.md

```diff
@@ -0,0 +1,29 @@
+# Chainlink External Adapter for Harmony
+This adapter is built to fulfill Chainlink oracle requests.
+## Configurations
+The adapter uses the following environment variables
+* HMY_NODE_URL: a URL to a JSON-RPC (HTTP RPC) node on Harmony, default: `https://api.s0.t.hmny.io`
+* HMY_CHAIN_ID: Harmony network chain id, default: `1 (mainnet)`
+* GAS_LIMIT: gaslimit to use, default: `6721900`
+* PRIVATE_KEY: user private key for signing the contract transaction
+
+## Input Params
+
+* address: the oracle contract to fulfill the request on
+* functionSelector: the fulfillment function selector
+* dataType: the data type, e.g., `uint256`, `bytes32`, `int256`
```

Comment:
Is the bytes32 encoding any different from the standard EVM bytes32 encoding? We try to leave most of these things to the adapter before the write adapter, meaning we usually expect the input here to already be encoded.

Reply:
@boxhock it is standard evm bytes32. just followed ethwrite-adapter: https://github.com/smartcontractkit/ethwrite-adapter/blob/master/index.js#L9

Reply:
removed.